protostar/cairo_migrator.py

```diff
@@ -0,0 +1,45 @@
+from logging import Logger
+from pathlib import Path
+from typing import List
+
+from starkware.cairo.lang.compiler.ast.formatting_utils import FormattingError
+from starkware.cairo.lang.migrators.migrator import parse_and_migrate, MIGRATE_FUNCTIONS
+
+from protostar.protostar_exception import ProtostarException
+
+
+class CairoMigrator:
+    def __init__(self, logger: Logger, single_return_functions: bool = True):
```

Comment:
Defaults for booleans are `False` in the entire project.

Reply:
`single_return_functions` is never used and could be removed from `CairoMigrator`.

Reply:
deleted